business_requirement_deliverable_categ/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================================
+Business Requirement Deliverable Task Category
+==============================================
+
+This module extends module business_requirement_deliverable and business_requirement_project.
+
+In business requirement if create a new business requirement resource line and if it is for a "Task" resource you could add the task category.
+
+Business requirement could generate a project that also will create task. This module will add the task category from business requirement resource line.
+
+Business requirement resource line also has the task name. This module merge the task name with the Business Requirement name like "BR_name-Task_name".
+
+Module business_requirement_deliverable:
+
+* Add field task_categ_id
+
+
+Module business_requirement_project:
+
+* Prefill the task category with business.requirement.resource.line "task_categ_id" 
+* Merge task name with business.requirement "name" and business.requirement.resource.line "task_name"
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (project)
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```

Comment:
This module is aimed at OCA (as this a fork from the main). You need to use OCA data and README template for the bottom part.